README.md

```diff
@@ -7,18 +7,18 @@
   - [🛠 Built With](#🛠-built-with)
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
-  - [🚀 Live Demo](#🚀-live-demo)
-- [💻 Getting Started](#💻-getting-started)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
   - [Setup](#setup)
   - [Prerequisites](#prerequisites)
   - [Install](#install)
   - [Usage](#usage)
   - [Run tests](#run-tests)
-- [👥 Authors](#👥-authors)
-- [🔭 Future Features](#🔭-future-features)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
 - [🤝 Contributing](#🤝-contributing)
```

Comment:
Great job @ClintonYounge 🤝

- [ ] Kindly fix the broken issues link ti make your readme file more professional.